### PR TITLE
New changelogs reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -5,12 +5,12 @@ Not yet released.
 
 * Improved user pages and added listing of users.
 * Dropped support for migrating from 3.x releases, migrate through 4.1 or 4.0.
-* Added exports in several monolingual formats.
+* Added exports for several monolingual formats.
```
Comment (nijel) [inline docs/changes.rst:8]:
```suggestion
* Added exports into several monolingual formats.
```

Comment (comradekingu) [reply docs/changes.rst:8]: What are "exports" here?

Comment (nijel) [reply docs/changes.rst]: Downloading translations in other format, see https://docs.weblate.org/en/latest/user/files.html. The list is now extended to include several monolingual formats (Android, JSON, iOS,...).

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**